res/values/strings.xml

```diff
@@ -347,13 +360,23 @@
     <!-- mailing lists -->
     <string name="mailing_list">Mailing list</string>
     <string name="ask_show_mailing_list">Show mailing list \"%1$s\"?</string>
-
+    <string name="mailing_list_profile_info">These settings (chat name and image) apply to this device only.</string>
```

Comment:
"chat name" is a term that is used nowhere else yet, so maybe not introduce that here.

as already discussed, i do not like the brackets here much. my suggestion would be 

```suggestion
    <string name="mailing_list_profile_info">Mailing list name and image changes apply to this device only.</string>
```

Reply:
Users might not think of those changeable properties as settings so it could be confusing. That's why I insist on the clarification.